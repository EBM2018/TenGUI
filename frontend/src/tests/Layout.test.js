import React from 'react';
import renderer from 'react-test-renderer';
import Layout from '../react/Layout/Layout.js';
import CheckBox from '../react/Widgets/Activity/CheckBox';

test('derp', () => {
  expect('True').toBeTruthy;
});
test('Layout renders correctly', () => {
  const component = renderer
    .create(<Layout children={<CheckBox id={1} reponse="42" fctChecked={() => 'This is a test'} />} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
