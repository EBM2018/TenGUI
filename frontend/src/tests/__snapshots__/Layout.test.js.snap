// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div>
  <div
    id="header"
  >
    <header>
      <div
        className="columns add-margin"
      >
        <div
          className="column"
        />
      </div>
      <div
        className="columns add-margin"
      >
        <div
          className="column is-2"
        >
          <figure
            className="image"
          >
            <img
              alt="TenuGUI"
              src="LogoTenuGUI.png"
            />
          </figure>
        </div>
        <div
          className="column is-8"
        />
        <div
          className="column"
        >
          <figure
            className="image is-64x64"
          >
            <img
              alt="MonProfil"
              src="../../../images/PhotoProfil.png"
            />
            <p>
              
            </p>
          </figure>
        </div>
      </div>
    </header>
  </div>
  <main>
    <input
      checked={true}
      onChange={[Function]}
      type="checkbox"
    />
    42
  </main>
</div>
`;
